docs/installation.md

```diff
@@ -5,6 +5,8 @@ To use the default installation, you need to have [docker installed](https://doc
 
 **Windows users** must have Docker Desktop running.
 
+You system user also needs read/write access to the directory that contains your config file; typically this is also the directory where you are running the opsconsole.
```

Comment:
Do we also need/want to mention the $HOME directory caveat here for Linux superusers?